lab_5_scrapper/scrapper.py

```diff
@@ -1,8 +1,73 @@
 """
 Crawler implementation
 """
+import concurrent.futures
```

Comment:
let's try to find more workarounds. does it take so much time on your local computer as well?